cli/uninstall/uninstall.go

```diff
@@ -226,25 +233,41 @@ func searchLatestVersion(linkName, binDst, headerDst string) (string, error) {
 		if util.Find(programsToSearch, programName) == -1 {
 			continue
 		}
+		isRightFormat, err := checkHashFormat(matches["ver"])
 
-		ver, err := version.Parse(matches["ver"])
 		if err != nil {
 			continue
 		}
-		if strings.Contains(programName, "tarantool") {
-			// Check for headers.
+
+		if isRightFormat && !strings.Contains(matches["ver"], ".") && !hashFound {
```

Comment:
I'm a little bit confusing by this condition. If `isRightFormat` is set to true, it means this is valid hash format according to line 236. If this is valid hash format, why do we need to check it does not contain dots?

Reply:
fixed